spdx-utils/src/main/kotlin/SpdxLicenseAliasMapping.kt

```diff
@@ -111,26 +111,7 @@ object SpdxLicenseAliasMapping {
         "w3cl" to W3C,
         "wtf" to WTFPL,
         "zope" to ZPL_2_1
-    ).also {
-        val keys = it.unzip().first.toMutableList()
-        val uniqueKeys = keys.distinct()
-        if (keys.size > uniqueKeys.size) {
-            uniqueKeys.forEach { uniqueKey -> keys.remove(uniqueKey) }
-            require(keys.isEmpty()) {
-                val quotedKeys = keys.map { "\"$it\"" }
-                "The following ${keys.size} keys are present in the same capitalization: $quotedKeys"
-            }
-        }
-    }.toMap().mapValues { (_, v) -> v.toExpression() }.let { caseSensitiveMap ->
-        caseSensitiveMap.toSortedMap(String.CASE_INSENSITIVE_ORDER).also { caseInsensitiveMap ->
-            if (caseSensitiveMap.size > caseInsensitiveMap.size) {
-                val difference = caseSensitiveMap.keys.subtract(caseInsensitiveMap.keys).map { "\"$it\"" }
-                require(difference.isEmpty()) {
-                    "The following ${difference.size} keys are present in different capitalizations: $difference"
-                }
-            }
-        }
-    }
+    ).checkForSameKeyCaptialization().mapValues { (_, v) -> v.toExpression() }.checkForDifferentKeyCaptialization()
```

Comment:
Now that it would be easy to do, should these checks be applied to the maps below as well?

Reply:
Yes, I was planning to do exactly that later, ideally in a way that does not require to convert the maps (and their "sum") multiple times to a case-insensitive map. We should probably merge your PR first, and then I'll look how to get that done.

Reply:
I'll skip that for now.